release-process.md

```diff
@@ -75,9 +44,37 @@ For example if you are cutting RC for 1.0.2, mark such issues as `FIXED` in 1.0.
 release, and change them to the current release.
 - Verify from `git log` whether they are actually making it in the new RC or not.
 
-The process of cutting a release candidate has been automated via the AMPLab Jenkins. There are 
-Jenkins jobs that can tag a release candidate and create various packages based on that candidate. 
-The recommended process is to ask the previous release manager to walk you through the Jenkins jobs.
+The process of cutting a release candidate has been partially automated via the AMPLab Jenkins. There are
+Jenkins jobs that can tag a release candidate and create various packages based on that candidate.
+
+
+At present the Jenkins jobs *SHOULD NOT BE USED* as they use a legacy shared key for signing.
+Instead much of the same release logic can be accessed in `dev/create-release/release-tag.sh` and `dev/create-release/release-build.sh`. The general order of creating a release using the scripts is:
+
+- Verify Jenkins test pass on your desired commit
+- Set the shell enviroment variables used by the scripts (run with "help" for details)
+- Verify your JAVA_HOME is set to the correct Java version (2.2+ Java 8, pre-2.2 Java 7)
+- You may find Felix's docker env useful - https://github.com/felixcheung/spark-build/blob/master/Dockerfile .
+- Ensure you have the required dependcies to build the docs `docs/README.md`
+- R, for some reason, requires e1071 to be installed as part of the packaging tests.
+- In addition R uses LaTeX for some things, and requires some additional fonts. On debian based systems you may wish to install `texlive-fonts-recommended` and `texlive-fonts-extra`.
+- Make sure you required Python packages for packaging (see `dev/requirements.txt`)
+- Tag the release candidate with `dev/create-release/release-tag.sh` (e.g. for creating 2.1.2 RC2 we did `ASF_USERNAME=holden ASF_PASSWORD=yoursecretgoeshere GIT_NAME="Holden Karau" GIT_BRANCH=branch-2.1 GIT_EMAIL="holden@us.ibm.com" RELEASE_VERSION=2.1.2 RELEASE_TAG=v2.1.2-rc2 NEXT_VERSION=2.1.3-SNAPSHOT ./dev/create-release/release-tag.sh`)
+- Package the release binaries & sources with `dev/create-release/release-build.sh package`
+- Create the release docs with `dev/create-release/release-build.sh docs`
+- For Spark versions prior to 2.1.2, change the SPARK_VERSION from X.Y.Z to X.Y.Z-rcA then run `dev/create-release/release-build.sh publish-release`.
+- Publish a snapshot to the Apache release repo `dev/create-release/release-build.sh publish-release`
+
+
+If the Jenkins jobs have been updated to support signing with your key you can look at the job required for a release are located in the <a href="https://amplab.cs.berkeley.edu/jenkins/view/Spark%20Release/">Spark Release Jobs</a> collection.
```

Comment:
is there a JIRA/ask tracking the change to Berkeley Jenkins process?

Reply:
SPARK-22055 & SPARK-22054 are the related JIRAs